weblate/trans/management/commands/benchmark.py

```diff
@@ -58,7 +58,9 @@ def handle(self, *args, **options) -> None:
             file_format=options["format"],
             project=project,
         )
-        profiler.runcall(component.after_save, True, False, False, False, True, True)
+        profiler.runcall(
+            component.after_save, True, False, False, False, False, True, True
+        )
```

Comment:
There should be no need for a change here:

```suggestion
        profiler.runcall(component.after_save, True, False, False, False, True, True)
```